packages/EasyApiToken/src/External/Auth0JwtDriver.php

```diff
@@ -106,6 +106,8 @@ public function decode(string $token): array
         $config = new Auth0V8SdkConfiguration([
             'domain' => $this->domain,
             'clientId' => 'client_id',
+            'strategy' => Auth0V8SdkConfiguration::STRATEGY_API,
```

Comment:
We need this after this commit https://github.com/auth0/auth0-PHP/commit/a1fd98f8569649717cc50e18fb92521091c1312e

Reply:
@alexndlm I think that would break applications that update the monorepo but not auth0

Reply:
Agree, we have to ensure that everything will work with version 8.0.

Reply:
@natepage these settings already exists in 8.0. From 8.3 these are required. Current changes should not impact apps.